tools/influx_data_manager/influx_data_manager.xml

```diff
@@ -0,0 +1,82 @@
+<tool id="influx_data_manager" name="Influx Data Manager: Handling influx_si data inputs in Galaxy workflows" version="@TOOL_VERSION@+galaxy0" profile="21.09">
+    <macros>
+        <token name='@TOOL_VERSION@'>1.0.0</token>
+    </macros>
+    <requirements>
+        <requirement type='package' version='@TOOL_VERSION@'>influx-si-data-manager</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    influx_data_manager
+    --physiofit $in_physiofit 
+    --isocor $in_isocor 
+    --linp $in_linp 
+    --netw $in_netw 
+    --mmet $in_mmet 
+    --cnstr $in_cnstr 
+    --tvar $in_tvar 
+    --opt $in_opt 
+    --log $out_log
+    #if $verbose == 'true':
+        -v
+    #end if 
+    2> $err_out
+    ]]></command>
+    <inputs>
+        <param type='data' format='tabular, txt' name='in_netw' label='netw' help='File containing network for experiments (.netw file)' optional='false'/>
```

Comment:
You can use `argument=--netw` instead of `name`.
And/Or use part of the help as name in complement with argument

Reply:
What do you mean by use part of the help as name?

Reply:
Oups, sorry, I mean `label`
`label` is displayed above the form, the help under.
Moreover, `argument` is automatically add to the help `(--netw)` 

So something, like that
```xml
<param type='data' format='tabular,txt' argument='--netw' label="File containing network for experiments (.netw file)" optional='false'/>
```

Reply:
Ahh I see. That seems clearer yes. I will implement that thanks